iothub/service/src/Exceptions/IotHubSuspendedException.cs

```diff
@@ -14,13 +14,15 @@ namespace Microsoft.Azure.Devices.Common.Exceptions
     [Serializable]
     public class IotHubSuspendedException : IotHubException
     {
+        private const string SuspendedException = "IotHubSuspendedException";
+
         /// <summary>
         /// Creates an instance of this class with a name of the suspended IoT hub
         /// and marks it as non-transient.
         /// </summary>
         /// <param name="iotHubName">The name of the IoT hub that has been suspended.</param>
         public IotHubSuspendedException(string iotHubName)
-            : base(Resources.IotHubSuspendedException.FormatInvariant(iotHubName))
+            : base(SuspendedException.FormatInvariant(iotHubName))
```

Comment:
Why does it need formatting?

Reply:
Member object name cannot be same as class name,

Reply:
I don't understand what you mean by that.

SuspendedException is a string with no format specifications. What is the effect of calling `FormatInvariant` with a parameter of `iotHubName`?

Reply:
Fixed

Reply:
Did you push the fix yet? I don't see it.

Reply:
https://github.com/Azure/azure-iot-sdk-csharp/pull/2621/commits/d55ddbc1f6a7f9c25e8219d0bf7a7cbcbb01bff1#diff-b76d036e576207ba57817f2ce9bb7dbc2f93dd8f217754988c83d99b318bac06